core/src/main/java/nl/nn/adapterframework/filesystem/ExchangeFileSystem.java

```diff
@@ -1214,48 +1232,62 @@ public void setPassword(String password) {
 		setClientSecret(password);
 	}
 
-	/** If empty, all mails are retrieved. If set to <code>NDR</code> only Non-Delivery Report mails ('bounces') are retrieved */
+	/**
+	 * If empty, all mails are retrieved. If set to <code>NDR</code> only Non-Delivery Report mails ('bounces') are retrieved
+	 */
 	public void setFilter(String filter) {
 		this.filter = filter;
 	}
 
 
-	/** proxy host */
+	/**
+	 * proxy host
+	 */
```

Comment:
Ik denk dat het mooier (?) is wanneer het maar 1 regel is, de description ook op 1 regel te houden.

Reply:
Niet mijn eigen voorkeur, maar zoals u zegt, meneer!
😉

Reply:
Dat heeft ook mijn voorkeur